spec.html

```diff
@@ -37,6 +37,7 @@ <h1>Promise.withResolvers ( )</h1>
           1. Let _C_ be the *this* value.
           1. Let _promiseCapability_ be ? NewPromiseCapability(_C_).
           1. Let _obj_ be OrdinaryObjectCreate(%Object.prototype%).
+          1. Perform ! CreateDataPropertyOrThrow(_obj_, "[Symbol.iterator]", _promiseCapability_.[[Iterator]]).
```

Comment:
this slot isn’t a thing on promise capability records. Also,
```suggestion
          1. Perform ! CreateDataPropertyOrThrow(_obj_, %Symbol.iterator%, _promiseCapability_.[[Iterator]]).
```

Reply:
`@@iterator`

Reply:
ah right, i forgot https://github.com/tc39/ecma262/pull/1314 hasn't landed yet. either way this is correct as-is, it's just not matching the current (but soon to be removed) convention.

Reply:
Honestly, I'm simply not sure what's require here 🤷🏽‍♂️